src/views/js/utilities.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * @author colsen@sfy.co.za
+ * @since 14-Jul-17.
+ */
+
+function getStringObjectFromCookie(objectKey) {
+    var allKeysAndValues = document.cookie.split(";");
+
+    for (var i = 0; i < allKeysAndValues.length; i++) {
+        var keyAndValue = allKeysAndValues[i];
+        var keyAndValueAsArray = keyAndValue.split("=");
+        var keyStr = keyAndValueAsArray[0].trim();
+
+        if (keyStr === objectKey) {
+            return keyAndValueAsArray[1];
+        }
+    }
+    setStringObjectOnCookie(objectKey, "");
+    return getStringObjectFromCookie(objectKey);
+}
+
+function setStringObjectOnCookie(objectKey, objectValue) {
+    var allKeysAndValues = document.cookie.split(";");
+
+    if (!(allKeysAndValues > 1)) {
+        newValue = objectKey + "=" + objectValue;
+        document.cookie = newValue;
+        return;
+    }
+
+    for (var i = 0; i < allKeysAndValues.length; i++) {
+        var keyAndValue = allKeysAndValues[i];
+        var keyAndValueAsArray = keyAndValue.split("=");
+
+        var keyStr = keyAndValueAsArray[0].trim();
+        if (keyStr === objectKey) {
+            var newValue = objectKey + "=" + objectValue;
+            document.cookie = document.cookie.replace(keyAndValue, newValue);
+            return;
+        }
+    }
+
+    newValue = objectKey + "=" + objectValue;
+    document.cookie = document.cookie + ";" + newValue;
+}
+
+function readSettings() {
+    var cookies = document.cookie.split(";");
+
+    for (var i = 0; i < cookies.length; i++) {
+        var cookieValue = cookies[i].split("=");
+        var name = cookieValue[0].trim();
+
+        if (name == "color") {
+            var color = cookieValue[1];
+            switch (color) {
+                case "green":
+                    document.getElementById("blueTheme").disabled = true;
+                    document.getElementById("redTheme").disabled = true;
+                    document.getElementById("greenTheme").disabled = false;
+                    break;
+                case "blue":
+                    document.getElementById("redTheme").disabled = true;
+                    document.getElementById("greenTheme").disabled = true;
+                    document.getElementById("blueTheme").disabled = false;
+                    break;
+                case "red":
+                    document.getElementById("blueTheme").disabled = true;
+                    document.getElementById("greenTheme").disabled = true;
+                    document.getElementById("redTheme").disabled = false;
+                    break;
+            }
+        }
+
+        if (name == "font") {
+            var font = cookieValue[1];
+            switch (font) {
+                case "sans-serif":
```

Comment:
Rather assign "document.body.style.fontFamily with the value of font. This will prevent the possibility of typos in multiple case statements